src/lib.rs

```diff
@@ -8,14 +8,15 @@
 // option. This file may not be copied, modified, or distributed
```

Comment:
Could you remove the license header (since you changed the licensing details to match up with the main repo, which also doesn't have headers).